Add H-K lettered article stubs from glossary (#3828)
ppy/osu-wiki: wiki/Beatmap_Editor/Kiai_Time/en.md

---
tags:
  - timing section
---

# Kiai Time

> *For regulations surrounding kiai time, see the [Ranking Criteria](/wiki/Ranking_Criteria).*

Reviewer: > Is kiai time a proper noun or not? this article is inconsistent in its capitalisation

Reviewer 2: I don't think it is. I've updated the article

> **Kiai Time** is a term given for a "special" timing section to help emphasize a part of a [beatmap](/wiki/Beatmaps). A player can tell if a part of a beatmap has a kiai timing section by seeing added visual effects such as flashing beats, flying stars, and fountains.

Reviewer: ```suggestion
**Kiai Time** is a term given for a "special" timing section to help emphasize a part of a [beatmap](/wiki/Beatmaps). Players can tell if a part of a beatmap has a kiai timing section if they see added visual effects in the background (e.g., flashing beats, flying stars, and fountains).
```
sounds nicer imo

> While kiai time is active all [hit objects](/wiki/Hit_Objects) will flash at the rhythm of the [main beat](/wiki/Beatmap_Editor/Timing) and stars will explode from the cursor when a [hit circle](/wiki/Hit_Objects) is hit or will fall from the cursor when a [slider](/wiki/Hit_Objects) or [spinner](/wiki/Hit_Objects) is followed.

Reviewer: ```suggestion
While kiai time is active, all [hit objects](/wiki/Hit_object) will flash at the rhythm of the [main beat](/wiki/Beatmap_Editor/Timing) and stars will explode from the cursor when a [hit circle](/wiki/Hit_object/Hit_circle) is hit. Otherwise, they will fall from the cursor when a [slider](/wiki/Hit_object/Slider) or [spinner](/wiki/Hit_object/Slider) is followed.
```

## Beatmapping

> Kiai time is most commonly used for the "strongest part of a song" which usually is the chorus. Since kiai sections are the strongest part of a song it makes sense that they are slightly more challenging than the rest of the mapset to follow the song.

Reviewer: ```suggestion
Kiai time is most commonly used for the "strongest part of a song" which is usually the chorus. Since kiai sections are the strongest part of a song, they are typically slightly more challenging than the rest of the [beatmapset](/wiki/Beatmapsets) to follow the song.
```

## osu!taiko

> On [osu!taiko](/wiki/Game_Modes/osu!taiko) maps kiai sections give 20% more [score](/wiki/Score).

Reviewer: ```suggestion
On [osu!taiko](/wiki/Game_Modes/osu!taiko) beatmaps, kiai sections give 20% more [score](/wiki/Score).
```
